<?xml version="1.0" encoding="UTF-8"?>
<testng-results ignored="0" total="1" passed="0" failed="1" skipped="0">
  <reporter-output>
    <line>
      <![CDATA[AddNomineeTest script execution start]]>
    </line>
    <line>
      <![CDATA[Testscript Execution failed]]>
    </line>
  </reporter-output>
  <suite started-at="2023-01-12T17:58:18 IST" name="Suite" finished-at="2023-01-12T17:58:40 IST" duration-ms="21877">
    <groups>
    </groups>
    <test started-at="2023-01-12T17:58:18 IST" name="Test" finished-at="2023-01-12T17:58:40 IST" duration-ms="21877">
      <class name="Nominee.Nominee_Add_NomineeTest">
        <test-method is-config="true" signature="connectToDB()[pri:0, instance:Nominee.Nominee_Add_NomineeTest@399f45b1]" started-at="2023-01-12T17:58:18 IST" name="connectToDB" finished-at="2023-01-12T17:58:18 IST" duration-ms="29" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- connectToDB -->
        <test-method is-config="true" signature="configBC(java.lang.String)[pri:0, instance:Nominee.Nominee_Add_NomineeTest@399f45b1]" started-at="2023-01-12T17:58:20 IST" name="configBC" finished-at="2023-01-12T17:58:35 IST" duration-ms="15139" status="PASS">
          <params>
            <param index="0">
              <value>
                <![CDATA[Chrome]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- configBC -->
        <test-method is-config="true" signature="loginToApp()[pri:0, instance:Nominee.Nominee_Add_NomineeTest@399f45b1]" started-at="2023-01-12T17:58:35 IST" name="loginToApp" finished-at="2023-01-12T17:58:37 IST" duration-ms="1365" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- loginToApp -->
        <test-method signature="AddNominee()[pri:0, instance:Nominee.Nominee_Add_NomineeTest@399f45b1]" started-at="2023-01-12T17:58:37 IST" name="AddNominee" finished-at="2023-01-12T17:58:39 IST" duration-ms="2359" status="FAIL">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[null]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: null
at org.testng.Assert.fail(Assert.java:99)
at org.testng.Assert.fail(Assert.java:106)
at Nominee.Nominee_Add_NomineeTest.AddNominee(Nominee_Add_NomineeTest.java:38)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:567)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:598)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:173)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:824)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:146)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1507)
at org.testng.TestRunner.privateRun(TestRunner.java:794)
at org.testng.TestRunner.run(TestRunner.java:596)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:377)
at org.testng.SuiteRunner.access$000(SuiteRunner.java:28)
at org.testng.SuiteRunner$SuiteWorker.run(SuiteRunner.java:418)
at org.testng.internal.thread.ThreadUtil.lambda$execute$0(ThreadUtil.java:64)
at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628)
at java.base/java.lang.Thread.run(Thread.java:830)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
            <line>
              <![CDATA[AddNomineeTest script execution start]]>
            </line>
            <line>
              <![CDATA[Testscript Execution failed]]>
            </line>
          </reporter-output>
        </test-method> <!-- AddNominee -->
        <test-method is-config="true" signature="logoutOfApp()[pri:0, instance:Nominee.Nominee_Add_NomineeTest@399f45b1]" started-at="2023-01-12T17:58:39 IST" name="logoutOfApp" finished-at="2023-01-12T17:58:40 IST" duration-ms="355" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- logoutOfApp -->
        <test-method is-config="true" signature="closeBrowser()[pri:0, instance:Nominee.Nominee_Add_NomineeTest@399f45b1]" started-at="2023-01-12T17:58:40 IST" name="closeBrowser" finished-at="2023-01-12T17:58:40 IST" duration-ms="609" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- closeBrowser -->
        <test-method is-config="true" signature="closeDB()[pri:0, instance:Nominee.Nominee_Add_NomineeTest@399f45b1]" started-at="2023-01-12T17:58:42 IST" name="closeDB" finished-at="2023-01-12T17:58:42 IST" duration-ms="1" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- closeDB -->
      </class> <!-- Nominee.Nominee_Add_NomineeTest -->
    </test> <!-- Test -->
  </suite> <!-- Suite -->
</testng-results>
